src/calendar/tests/unit/MonthPicker.ts

```diff
@@ -229,7 +233,7 @@ registerSuite({
 
 		// enter key
 		closed = false;
-		widget.sendEvent('keydown', {
+		widget.sendEvent<TestEventInit>('keydown', {
```

Comment:
need to see if there is a way to make this easier... _sigh_ works for now though